src/mobo_qm9.py

```diff
@@ -79,6 +84,26 @@ def get_surrogate_model(self, X, y):
         returns:
             model: Surrogate model for the MOBOQM9 model.
         """
+        X_scaled = Standardize(X)
+        train_X = torch.tensor(X_scaled, dtype=torch.float32)
+        train_Y = torch.tensor(y, dtype=torch.float32)
+        likelihood = GaussianLikelihood()
+
+        if kernel_type == 'RBF':
+            kernel = RBFKernel()
+        elif kernel_type == 'Matern':
+            kernel = MaternKernel()
+        elif kernel_type == 'Tanimoto':
+            kernel = TanimotoKernel()
+        else:
+            raise ValueError("Unsupported kernel type. Supported types are 'RBF', 'Matern', and 'Tanimoto'.")
+
+        model = SingleTaskGP(train_X, train_Y, likelihood=likelihood, kernel=kernel)
```

Comment:
This is only for single target optimization, can you use `ModelListGP` for multi-target?